Purchases/Networking/Operations/Handling/SubscriberAttributeHandler.swift

```diff
@@ -22,28 +22,28 @@ class SubscriberAttributeHandler {
     }
 
     func handleSubscriberAttributesResult(statusCode: HTTPStatusCode,
-                                          response: [String: Any]?,
-                                          error: Error?,
+                                          response: Result<[String: Any], Error>,
                                           completion: SimpleResponseHandler) {
-        if let error = error {
-            completion(ErrorUtils.networkError(withUnderlyingError: error))
-            return
-        }
-
-        let responseError: Error?
-
-        if let response = response, !statusCode.isSuccessfulResponse {
-            let extraUserInfo = self.userInfoAttributeParser
-                .attributesUserInfoFromResponse(response: response, statusCode: statusCode)
-            let backendErrorCode = BackendErrorCode(code: response["code"])
-            responseError = ErrorUtils.backendError(withBackendCode: backendErrorCode,
-                                                    backendMessage: response["message"] as? String,
-                                                    extraUserInfo: extraUserInfo as [NSError.UserInfoKey: Any])
-        } else {
-            responseError = nil
-        }
-
-        completion(responseError)
+        let result: Result<[String: Any], Error> = response
+            .mapError {
+                ErrorUtils.networkError(withUnderlyingError: $0)
```

Comment:
This line is basically repeated in all operations, so it'll be moved to `HTTPClient` too.